mkt/api/fields.py

```diff
@@ -432,8 +412,13 @@ class SemiSerializerMethodField(serializers.SerializerMethodField):
     Used for fields serialized with a method on the serializer but who
     need to handle unserialization manually.
     """
-    def field_from_native(self, data, files, field_name, into):
-        into[field_name] = data.get(field_name, None)
+    def __init__(self, method_name=None, **kwargs):
+        # Intentionally skipping SerializerMethodField.__init__.
```

Comment:
`SerializerMethodField.__init__` sets the field read-only. I'll clarify the comment.